include/swift/AST/Decl.h

```diff
@@ -819,7 +819,11 @@ class alignas(1 << DeclAlignInBits) Decl {
 
   // Make vanilla new/delete illegal for Decls.
   void *operator new(size_t Bytes) = delete;
+
+  // Work around MSVC error: attempting to reference a deleted function.
```

Comment:
I've submitted a connect bug for this, at https://connect.microsoft.com/VisualStudio/feedback/details/3116505